### ISSUE TITLE
WCAG failing on empty alt images when there's accompanying text in a link

### ISSUE THREAD

**Issue by anselmbradford** [cfpb/node-wcag#40]
According to https://achecker.ca/checker/suggestion.php?id=7, "img element cannot have alt attribute value of null or whitespace if the img element is contained by an A element and **there is no other link text.**"

See https://github.com/cfpb/cfgov-refresh/pull/3647. The images don't have alt text but do have accompanying link text, yet wcag still fails those images.
